ecma402/README.md

```diff
@@ -1,9 +1,9 @@
 # [ECMAScript Internationalization API Specification](https://github.com/tc39/ecma402) proposals
 
- - [Stage 0 Proposals](stage-0-proposals.md)
- - [Finished Proposals](finished-proposals.md)
+- [Stage 0 Proposals](stage-0-proposals.md)
+- [Finished Proposals](finished-proposals.md)
```

Comment:
i'll revert all these unrelated changes. please make sure your editor isn't blindly running prettier on all files - formatters should only run in projects that have them set up.